src/simperium/bucket.js

```diff
@@ -303,7 +308,7 @@ Bucket.prototype.getVersion = function( id, callback ) {
  */
 Bucket.prototype.touch = function( id, callback ) {
 	const task = this.storeAPI.get( id )
-		.then( object => this.update( object.id, object.data ) );
+		.then( object => this.update( { id: object.id, data: object.data } ) );
```

Comment:
```suggestion
		.then( object => this.update( object.id, object.data ) );
```

Reply:
Whoops, already pushed in 77df264554cde8cfec1c78f4dd10000eb49dbe3c